src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class WorkWithFile {
 
+    private int saply = 0;
+    private int buy = 0;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        getRead(fromFileName);
+        getWrite(toFileName);
+    }
+
+    public void getRead(String fromFileName) {
+        try (BufferedReader bufferedReader = Files.newBufferedReader(Paths.get(fromFileName))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] words = line.split(" ");
+
+                for (String word : words) {
+                    if (word.startsWith("s")) {
+                        saply += Integer.parseInt((word.replaceAll("\\D", "")));
+                    }
+                    if (word.startsWith("b")) {
+                        buy += Integer.parseInt((word.replaceAll("\\D", "")));
+                    }
+                }
+            }
+
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file");
+        }
+    }
+
+    public void getWrite(String toFileName) {
```

Comment:
```suggestion
    public void getWrite(String toFileName, String report) {
```
так метод буде більше реюзабельний за потреби